extension/src/triggers.rs

```diff
@@ -0,0 +1,56 @@
+use pgx::*;
+
+#[pg_extern]
+fn timescale_analytics_probe() {
+
+}
+
+extension_sql!(r#"
+CREATE OR REPLACE FUNCTION disallow_experimental_dependencies()
+  RETURNS event_trigger
+ LANGUAGE plpgsql
+  AS $$
+DECLARE
+  guc_set TEXT;
+  cmd record;
+  experimental_schema_id oid;
+BEGIN
+
+  guc_set := current_setting('timescale_analytics_install_experimental', true);
+  IF guc_set IS NOT NULL AND guc_set = 'on' THEN
+    RETURN;
+  END IF;
+
+  SELECT oid schema_oid
+  INTO experimental_schema_id
+  FROM pg_catalog.pg_namespace
+  WHERE nspname='timescale_analytics_experimental'
+  LIMIT 1;
+
+  FOR cmd IN
```

Comment:
Seems slightly weird to me to do this in a loop rather than an IF block with an exists? But it works I guess.

Reply:
fixed